server.go

```diff
@@ -2914,30 +2914,44 @@ func (s *server) announceChanStatus(op wire.OutPoint, disabled bool) error {
 
 // fetchLastChanUpdateByOutPoint fetches the latest update for a channel from
 // our point of view.
-func (s *server) fetchLastChanUpdateByOutPoint(op wire.OutPoint) (*lnwire.ChannelUpdate, error) {
+func (s *server) fetchLastChanUpdateByOutPoint(op wire.OutPoint) (
```

Comment:
Nit: the comment should read "either edge is not present". From my reading the prior code would possibly panic if one of the edges weren't advertised, while this will return with an error if we can't find our edge.